ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.17-nightly-x86.yaml

```diff
@@ -22,8 +22,8 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: control-plane-ipsec-120nodes
-  cron: 0 4 11,25 * *
+- always_run: false
+  as: control-plane-ipsec-120nodes
```

Comment:
Is there a reason here you're turning off this workload? I didnt add this workload but from the title of the PR I would think that you wouldnt edit other items

Reply:
we have been getting cloud quota issues with `azure-qe` profile lately, @svetsa-rh suggested to have a new perfscale profile for such large scale test so meanwhile we decided to turn this off - [thread](https://redhat-internal.slack.com/archives/C05CDC19ZKJ/p1720807013727689?thread_ts=1720715581.463399&cid=C05CDC19ZKJ) 
Let me know if you suggest to re-enable it.

failed [jobs](https://prow.ci.openshift.org/view/gs/test-platform-results/logs/periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-azure-4.17-nightly-x86-control-plane-ipsec-120nodes/1811771025721921536)

Reply:
I am fine to make it in a separate PR if you prefer that approach.

Reply:
no problem, just wasnt clear why this was turning things off

Reply:
Updated description, thanks.